composer.json

```diff
@@ -21,11 +21,20 @@
         "sensio/framework-extra-bundle": "~3.0,>=3.0.2",
         "sensio/generator-bundle": "~2.3",
         "incenteev/composer-parameter-handler": "~2.0",
-        "friendsofsymfony/user-bundle": "2.0.*@dev",
         "components/bootstrap": "~3.3",
         "components/jquery": "~2.1",
         "components/jqueryui": "~1.11"
     },
+    "require-dev": {
+        "behat/behat": "~3.0.14",
+        "behat/mink": "1.6.*",
+        "behat/symfony2-extension": "~2.0@dev",
+        "behat/mink-extension": "~2.0@dev",
+        "behat/mink-browserkit-driver": "~1.1@dev",
+        "phpspec/phpspec": "~2.1.0-RC3",
```

Comment:
Actually all dependencies from require-dev can be used without @dev as all of them has stable releases.